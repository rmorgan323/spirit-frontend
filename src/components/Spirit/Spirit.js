import React from 'react';
import { landing } from '../../language/language';
import './Spirit.css';

const Spirit = () => {
  return (
    <div className="Spirit">
      <img
        className="homepage-wordmark"
        src="/assets/Spirit-Homepage-Wordmark-Purple.png"
        alt="Spirit-Wordmark-with-Tagline"
      />
      <h4 className="homepage-brief-descriptor">{landing.descriptor}</h4>
    </div>
  );
};

export default Spirit;
