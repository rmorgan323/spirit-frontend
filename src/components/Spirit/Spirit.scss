@import '../../variables.scss';

.Spirit {
  @include primary-content;
  display: flex;
  flex-direction: column;

  .homepage-wordmark {
    align-self: center;
    width: 500px;
  }

  .homepage-brief-descriptor {
    align-self: center;
    max-width: 550px;
    text-align: center;
  }
}