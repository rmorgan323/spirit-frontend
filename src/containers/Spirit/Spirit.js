import React from 'react';
import './Spirit.css';

const Spirit = () => {
  return (
    <div className="Spirit">
      <img
        className="homepage-wordmark"
        src="/assets/Spirit-Homepage-Wordmark-Purple.png"
        alt="Spirit-Wordmark-with-Tagline"
      />
      <h4 className="homepage-brief-descriptor">
        A clinical reasoning model for ensuring best practice in pediatric
        occupational therapy.
      </h4>
    </div>
  );
};

export default Spirit;
