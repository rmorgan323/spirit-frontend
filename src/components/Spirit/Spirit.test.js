import React from 'react';
import Spirit from './Spirit';
import { shallow } from 'enzyme';

describe('Spirit tests', () => {
  it('Should match snapshot', () => {
    const renderedApp = shallow(<Spirit />);

    expect(renderedApp).toMatchSnapshot();
  });
});
