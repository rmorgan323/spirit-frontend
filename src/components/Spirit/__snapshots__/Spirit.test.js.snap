// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spirit tests Should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Spirit />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="Spirit-Wordmark-with-Tagline"
          className="homepage-wordmark"
          src="/assets/Spirit-Homepage-Wordmark-Purple.png"
        />,
        <h4
          className="homepage-brief-descriptor"
        >
          A clinical reasoning model for ensuring best practice in pediatric occupational therapy.
        </h4>,
      ],
      "className": "Spirit",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Spirit-Wordmark-with-Tagline",
          "className": "homepage-wordmark",
          "src": "/assets/Spirit-Homepage-Wordmark-Purple.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "A clinical reasoning model for ensuring best practice in pediatric occupational therapy.",
          "className": "homepage-brief-descriptor",
        },
        "ref": null,
        "rendered": "A clinical reasoning model for ensuring best practice in pediatric occupational therapy.",
        "type": "h4",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="Spirit-Wordmark-with-Tagline"
            className="homepage-wordmark"
            src="/assets/Spirit-Homepage-Wordmark-Purple.png"
          />,
          <h4
            className="homepage-brief-descriptor"
          >
            A clinical reasoning model for ensuring best practice in pediatric occupational therapy.
          </h4>,
        ],
        "className": "Spirit",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Spirit-Wordmark-with-Tagline",
            "className": "homepage-wordmark",
            "src": "/assets/Spirit-Homepage-Wordmark-Purple.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "A clinical reasoning model for ensuring best practice in pediatric occupational therapy.",
            "className": "homepage-brief-descriptor",
          },
          "ref": null,
          "rendered": "A clinical reasoning model for ensuring best practice in pediatric occupational therapy.",
          "type": "h4",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
